src/ServiceControl.Audit/Infrastructure/Bootstrapper.cs

```diff
@@ -83,6 +84,10 @@ void Initialize()
             var rawEndpointFactory = new RawEndpointFactory(settings, transportSettings, transportCustomization);
             containerBuilder.RegisterInstance(rawEndpointFactory).AsSelf();
 
+            var metrics = new Metrics();
```

Comment:
Is there any reason why we decided to have our own metrics instead of something like

https://docs.microsoft.com/en-us/dotnet/core/diagnostics/event-counters?

this should also be available on .NET Standard if I'm not mistaken

Reply:
I found the answer. When we started the Raven 5 branch SC was on `net462` so using EventCounters was not an option.